### PR TITLE
Creating register page, login page, onboarding page, and setting routes. Interface only

### PR DESCRIPTION
**Pull request by ajopi**
# Description
In this Commit i just created 3 pages, Login, Register and Onboarding without any function just interface only. it's not done yet, need improvement in background still waiting designer team for other option of assets. Please comment for any further revision, i'll fix it as soon as possible. 

JSON contract for API requirement: 
*1. Endpoint for Login*
request parameter: 
```json
{ "email" : "string",
   "password" : "string" }
```

*2. Endpoint for Register*
request parameter: 
```json
{ "name" : "string",
   "username" : "string" 
   "email": "string"
   "password": "string" }
```

## Task reference
All refrences based on figma design
> **1:** Create login page. [Task Reference](https://www.figma.com/file/XI7O8r49rFJNGH64hu8sbp/Guruku-UI-Team?node-id=1%3A6)
>**2.** Create register page [Task Reference](https://www.figma.com/file/XI7O8r49rFJNGH64hu8sbp/Guruku-UI-Team?node-id=1%3A6)
>**2.** Create onboarding page [Task Reference](https://www.figma.com/file/XI7O8r49rFJNGH64hu8sbp/Guruku-UI-Team?node-id=1%3A6)

## Type of change

Please make any relevant options for the progress of your changes.

> Example for ceating login page task :
> - [x] Make login page
> - [x] Make register page
> - [x] Make onbiarding page
> - [ ] Make form validation for email
> - [ ] Make form validation for password
> - [ ] Make form validation for empty form

# How has this been tested?
all of the pages are not been tested yet.
- [ ] Unit Test
- [ ] Postman Lesponse
- [ ] User Interface Local
- [ ] Staging
- [ ] Production

## Testing screenshot
Register:
image

Login:
image

Onboarding:
image

### Before my change
before my change it's just a default project from react app

### After my change
3 pages that i have been explain before

# Checklist:
- [x] My changes do not affect any errors in existing features
- [ ] My changes were up-to-date to the main branch. Perform `git pull --rebase origin main` to update your working branch
- [x] No conflict in my changes
- [ ] Any conflict are resolved
- [ ] My code follows [the style guidelines](https://) of this project
- [ ] I have performed a self-review of my code
- [ ] I've made [clear documentation](https://) as the style guide in my code
- [ ] I have provided  all relevant screenshots
- [ ] My changes generate no new warnings
- [ ] My changes generate no error
- [ ] I have added tests that prove my fix/update is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Approve by at least  2 or more tech member
- [ ]  Approve by the project owner
- [ ] Using all relevant labels
- [ ] Pass all merge requirements

### CODE REVIEW

**Review by chamisfum — COMMENTED** [review]
Nama app bukan lagi `guruku` tapi `Hero Guru`

**Comment by chamisfum** [comment]
> Screenshot:
> 
> Login: image
> 
> SignUp: image
> 
> onboarding: image

Coba lihat PR description ini:
https://github.com/cifacademy/student_hero_guru_frontend/pull/13

seharusnya ga perlu dihapus dan tinggal edit saja @ajopi

File: guruku-student/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (chamisfum) [inline guruku-student/.gitignore:1]: .gitignore nya bisa ditaruh di root folder ga ya?

**Review by chamisfum — COMMENTED** [review]
Pastikan sudah menerapkan ide utama dari clean code

File: guruku-student/.gitignore (hunk 1 above)
Comment (ajopi) [reply guruku-student/.gitignore:1]: sepertinya gabisa mas, sudah bawaan dari create react app nya

Comment (chamisfum) [reply guruku-student/.gitignore:1]: harusnya bisa kok kamu taruh di root folder aja gpp cobain nanti pasti bisa

Comment (chamisfum) [reply guruku-student/.gitignore:1]: @ajopi sudah liat link pr 13?

Comment (chamisfum) [reply guruku-student/.gitignore:1]: https://github.com/cifacademy/student_hero_guru_frontend/blob/main/Guideline/how_to_make_a_pull_request.md

Boleh baca guideline ini

Comment (ajopi) [reply guruku-student/.gitignore:1]: aku coba commit lagi ini berati ya?

Comment (chamisfum) [reply guruku-student/.gitignore:1]: kemarin udah ada template keluar ga pas pertama pr?

Comment (chamisfum) [reply guruku-student/.gitignore:1]: kaya e ga perlu kommit tinggal edit aja pr decriptionya

Comment (ajopi) [reply guruku-student/.gitignore:1]: > kemarin udah ada template keluar ga pas pertama pr?

template keluar gimana mas?? sorry2 gapaham

Comment (ajopi) [reply guruku-student/.gitignore:1]: > kaya e ga perlu kommit tinggal edit aja pr decriptionya

buat pindah gitignore nya kan perlu commit???

Comment (ajopi) [reply guruku-student/.gitignore:1]: nama app nya juga aku ganti sudah mas

Comment (ajopi) [reply guruku-student/.gitignore:1]: sudah di benerin mas, coba di cek lagi

**Review by chamisfum — APPROVED** [review]

**Review by ChamdaniMust — APPROVED** [review]

**Pull request merged**